packages/adp-tooling/package.json

```diff
@@ -39,22 +43,30 @@
         "@sap-ux/logger": "workspace:*",
         "@sap-ux/system-access": "workspace:*",
         "@sap-ux/ui5-config": "workspace:*",
+        "express-rate-limit": "6.9.0",
```

Comment:
Any new module will require an admin review, please add a quick explanation to the PR for what you need these new modules.